### PR TITLE
Adds new valentine's text, plus valentine-themed arcade enemies.

### PR DESCRIPTION
**Pull request by spessbandit**
## About The Pull Request

Adds new valentines, removes some ones I deemed not funny enough, adds candy hearts too
Adds valentine-themed arcade enemies, adds extra words to the other holidays because why not.

## Why It's Good For The Game

the public (me) demands new valentine's content (so it doesn't get old)

## Changelog
:cl: bandit
add: New valentines and Valentines-themed arcade machines will arrive next month. Y-you have a valentine, right?
/:cl:

### CODE REVIEW

File: strings/valentines.json [hunk 1]
```diff
@@ -30,35 +30,37 @@
 		"The virologist is rogue, and the only cure is a kiss from you.",
 		"Would you spend some time in my upgraded sleeper?",
 		"You must be a silicon, because you've unbolted my heart.",
+		"There's no way you're rogue -- you're a fine piece of Asimov.",
 		"Are you Nar'Sie? Because there's nar-one else I sie.",
-		"If you were a taser, you'd be set to stunning.",
```
Comment (HomocidlePancakes) [inline strings/valentines.json]: Keep this one.

Comment (spessbandit) [reply strings/valentines.json]: tasers don’t stun anymore

Comment (HomocidlePancakes) [reply strings/valentines.json]: Exactly. That's why you should keep it.

Comment (spessbandit) [reply strings/valentines.json]: if I didn’t want these to correspond to in game stuff I’d just copy paste a random space pickup lines buzzfeed list

Comment (HomocidlePancakes) [reply strings/valentines.json]: It references the good ‘ol days, and besides, hybrid tasers still have a stun mode, even though the stunspheres don’t stun.

File: strings/valentines.json [hunk 2]
```diff
@@ -73,11 +75,27 @@
...
 		"Let's make like swarmers and reproduce.",
-		"If you were a drone, would you be free tonight?"
+		"If you were a drone, would you be free tonight?",
+		"Get the paramedics -- your body put me into crit!",
+		"Come to the incinerator and be my maint squeeze.",
+		"Come to the courtroom so we can spend some time alone.",
+		"I'm the gray tide -- may I take your hand tele?",
+		"If I were a clown car, would you come inside me?",
+		"If you were telecoms, would you go down on me?",
+		"Don't touch any potassium, I might get you wet.",
```
Comment (kriskog) [inline strings/valentines.json]: Bit much dont you think? The rest are are mostly corny pick up lines

**Comment by Qustinnus** [comment]
Suggestion corny pickup line: you make me feel round and full inside

File: strings/valentines.json (hunk 2 above)
Comment (spessbandit) [reply strings/valentines.json]: 18+

**Comment by plapatin** [comment]
i will support this but only if you add "It's Valentine's Day and you're playing video games all by yourself? God, you really ARE pathetic." to the arcade cabinet window where the title usually is

File: strings/arcade.json [hunk 3]
```diff
@@ -114,14 +149,28 @@
...
 		"Unicorn",
 		"Tunnel Clown",
 		"Vhakoid"
 	],
 
+	"rpg_enemy_valentines": [
+		"Aphrodite",
+		"Comdom",
+		"Ex-Boyfriend",
+		"Ex-Girlfriend",
```
Comment (RaveRadbury) [inline strings/arcade.json]: Waifu but no husbando?

File: strings/arcade.json [hunk 4]
```diff
@@ -114,14 +149,28 @@
...
+		"Maneater",
+		"Spess Casanova",
+		"Spess Don Juan",
+		"St. Valentine",
+		"Waifu"
```
Comment (RaveRadbury) [inline strings/arcade.json]: Waifu but no husbando?

File: strings/valentines.json [hunk 5]
```diff
@@ -73,11 +75,26 @@
...
+		"I'm the gray tide -- may I take your hand tele?",
+		"If I were a clown car, would you come inside me?",
+		"If you were telecoms, would you go down on me?",
+		"Your face gets me higher than omega weed.",
+		"Someone check the tachyon doppler array, because you just rocked my world.",
+		"I wish I were your family heirloom so I could be with you all the time.",
+		"How much damage does a toxins bomb do? Enough to break the ice.",
+		"Call me a gibtonite deposit, because I've got a lot of love to gib tonite.",
+		"I don't need an ion cannon, I've got my ion you.",
+		"I don't need the chef's cooking when you're such a snack!",
```
Comment (RaveRadbury) [inline strings/valentines.json]: This might read better as "I don't need to be a cook to know you're a snack!"

Comment (spessbandit) [reply strings/valentines.json]: I mean everyone can tell what's a snack, it has different in-game effects than chef food /overthinking

Comment (RaveRadbury) [reply strings/valentines.json]: OH! I misinterpreted the line. This is great. Nevermind.

File: strings/valentines.json (hunk 2 above)
Comment (kriskog) [reply strings/valentines.json]: I meant all three of these

**Comment by spessbandit** [comment]
@plapatin look at the comments on valentines.dm

File: strings/arcade.json (hunk 4 above)
Comment (spessbandit) [reply strings/arcade.json]: will fix this blatant sexism when I get home

**Comment by nemvar** [comment]
disappointed
This is so sad.

**Comment by spessbandit** [comment]
this is so sad alexa play "when you write it"

File: strings/valentines.json [hunk 6]
```diff
@@ -2,24 +2,23 @@
...
+		"I'm not the atmos tech, but I'm great at laying pipe.",
 		"Could I have all access... to your heart?",
-		"Call me the doctor, because I'm here to inspect your johnson.",
+		"Call me the medical doctor, because I'm here to inspect your johnson.",
 		"Quick, get the defibrillator! I saw you and my heart stopped.",
 		"I'm not a changeling, but you make my proboscis extend.",
 		"I just can't get EI NATH of you.",
-		"You must be a nuke op, because you make my heart explode.",
-		"Roses are red / Botany is a farm / Not being my Valentine / causes human harm.",
+		"Roses are red / Intents are disarm / Not being my Valentine / causes human harm.",
```
Comment (Dorsisdwarf) [inline strings/valentines.json]: OOC / IC
This description needs changing
Intent's a game term

Comment (spessbandit) [reply strings/valentines.json]: not really

Comment (spessbandit) [reply strings/valentines.json]: like you can only refer to a "round" in a game context, but you can have in-character, an intent to push someone

Comment (cacogen) [reply strings/valentines.json]: hu cares

**Pull request merged**